Manito/Manito/Presentation/Scene/DetailScene/Detail/ViewController/FriendListViewController.swift

```diff
@@ -0,0 +1,141 @@
+//
+//  FriendListViewController.swift
+//  Manito
+//
+//  Created by SHIN YOON AH on 10/19/23.
+//
+
+import Combine
+import UIKit
+
+final class FriendListViewController: UIViewController, Navigationable {
+    
+    enum Section: CaseIterable {
+        case main
+    }
+    
+    // MARK: - ui component
+    
+    private let friendListView: FriendListView = FriendListView()
+    
+    private var dataSource: UICollectionViewDiffableDataSource<Section, MemberInfo>?
+    private var snapShot: NSDiffableDataSourceSnapshot<Section, MemberInfo>?
+
+    // MARK: - property
+    
+    private var cancelBag: Set<AnyCancellable> = Set()
+    
+    private var viewModel: any BaseViewModelType
+    
+    // MARK: - init
+    
+    init(viewModel: any BaseViewModelType) {
+        self.viewModel = viewModel
+        super.init(nibName: nil, bundle: nil)
+    }
+    
+    @available(*, unavailable)
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    // MARK: - life cycle
+    
+    override func loadView() {
+        self.view = self.friendListView
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.setupNavigation()
+        self.configureNavigationBar()
+        self.configureDataSource()
+        self.bindViewModel()
+    }
+    
+    // MARK: - func
+    
+    private func configureNavigationBar() {
+        self.friendListView.configureNavigationBar(self)
+    }
+    
+    private func bindViewModel() {
+        let output = self.transformedOutput()
+        self.bindOutputToViewModel(output)
+    }
+
+    private func transformedOutput() -> FriendListViewModel.Output? {
+        guard let viewModel = self.viewModel as? FriendListViewModel else { return nil }
+        let input = FriendListViewModel.Input(
+            viewDidLoad: self.viewDidLoadPublisher
+        )
+        return viewModel.transform(from: input)
+    }
+
+    private func bindOutputToViewModel(_ output: FriendListViewModel.Output?) {
+        guard let output = output else { return }
```

Comment:
p4;

```suggestion
        guard let output else { return }
```
지난번처럼 이렇게 적어도 될 것 같아요!